scss/tech.scss

```diff
@@ -1,42 +1,29 @@
 $tech-logo-size: 100px;
 
 .techs-container {
-  display: flex;
-  flex-wrap: wrap;
-  margin-left: 100px;
+  display: grid;
+  margin: 0 auto;
+  width: 900px;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-columns: 1fr 1fr;
+  grid-row-gap: 50px;
+  grid-template-areas:
+    "techcol tech tech"
+    "techcol tech tech";
```

Comment:
template areas is not a good idea here, since you are not using different names in the rest of the classes. Just leave the template columns and rows